File: TD/TD1/Code/3.cpp
/**
 * Informer si le produit est poitif, négatif ou nul, sans le calculer
 */
#include <iostream>

int main()
{
  int a;
  int b;

  std::cout << "Donne moi premier entier : " << std::flush;
  std::cin >> a;
  std::cout << "Donne moi deuxième entier : " << std::flush;
  std::cin >> b;

  if ((a < 0 && b < 0) || (a > 0 && b > 0))
    std::cout << "Le produit est positif !" << std::endl;
  else if (a == 0 || b == 0)
    std::cout << "Le produit est nul !" << std::endl;
  else
    std::cout << "Le produit est négatif !" << std::endl;

  return 0;
}

File: TD/TD1/Code/Python/3.py
#!/usr/bin/python
# Informer si le produit est poitif, négatif ou nul, sans le calculer #
print('Donne moi premier entier : ', end='')
a = int(input())
print('Donne moi deuxième entier : ', end='')
b = int(input())

if (a > 0 and b > 0) or (a < 0 and b < 0):
    print('Le produit est positif !')
elif (a == 0 or b == 0):
    print('Le produit est nulle !')
else:
    print('Le produit est négatif !')
